packages/app-builder/src/services/editor/ast-editor.tsx

```diff
@@ -315,16 +317,56 @@ export function useRootOrAndChildValidation(stringPath: string) {
   ]);
 }
 
-export function useValidationStatus(stringPath: string) {
+export function useValidationStatus(
```

Comment:
Honestly I'm not super proud of my code here, but this is precisely part of what I'd like to look at together on Wed.
In the meantime, it works and I don't think it's fundamentally broken.

Reply:
Ok, let's discuss this on wed.

To keep track of it (depending on if we keep the path access or not):

You can push your good renaming proposition further :
1. rename path -> treePath
2. rename `usePath` -> `useTreePath`
3. rename `useParentPath` -> `useParentTreePath`
4. rename `stringPath`-> `stringTreePath` ?